get_category_items.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+
+"""
```

Comment:
Add name of the file after the quotes like [here](https://github.com/srish/MediaWiki-Action-API-Code-Samples/blob/master/clientlogin.py). Same goes for other two files too.